fdbrpc/protos/file_transfer.proto

```diff
@@ -0,0 +1,18 @@
+syntax = "proto3";
+
+package fdbrpc;
+
+service FileTransferService {
+    rpc DownloadFile(DownloadRequest) returns (stream DownloadChunk);
+}
+
+message DownloadRequest {
+    string file_name = 1;
+    int32 chunk_size = 2;
+    int32 first_chunk_index = 3;
```

Comment:
File size? checksum? Metadata on file?

Reply:
Added another rpc for that.

Reply:
What you thinking? Why not inline w/ the file request rather than do another rpc?

Reply:
Thought it was just cleaner this way as these operations are not really latency sensitive and bulk of work will be done by DownloadRequest.

Reply:
Two RPCs when we could do one seems like a basic. Suggest a follow-on to fix this.

Reply:
yeah, I can do that as follow up.